labs/solutions/dom/collapse/collapse.js

```diff
@@ -0,0 +1,7 @@
+(function() {
```

Comment:
This isn't a good solution I think it needs a rethink!

Reply:
Based on the HTML I think you're better off targeting the selector like so.

```js
const icons = document.querySelectorAll('.expand-collapse-btn')
```

Reply:
yep yep that is a lot cleaner agreed, i wanted this less nice solution as a solution to the challange section which requires students to know how event bubbling works.

I'll add in a cleaner solution as well so students can see both, are you ok with that or do you wanna remove the challenge entirely? (i didn't think the fact that it was non optimal would matter since it's 2041 and a lot of code the kids write is hacky but in a educational way)

Reply:
Hmmmm.  I’ll reread the question. Maybe just a tweak.